frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css

```diff
@@ -0,0 +1,28 @@
+table.treetable span.indenter {
```

Comment:
please add header for small files, such as in:
https://github.com/sw360/sw360portal/blob/master/frontend/sw360-portlet/src/main/webapp/css/search.css